### DISCUSSION TITLE
Video preview and what is a "live" video?

### DISCUSSION THREAD

**Discussion by blue-kaleidoscope** [photoprism/photoprism#2765] [Q&A] [answered]
Hi,

I realized by accident that photoprism is able to create a video preview when a video was recognized as "live" video:

https://user-images.githubusercontent.com/56046815/194335514-91c0227b-e7a4-46ab-99a1-05a891865db4.mp4

Now I have 2 questions:

- Is this also possible for normal videos that show the white play button instead of the live button?
- What is the purpose behind the live videos or the difference towards normal videos? Both behave the same with the only difference mentioned above. Is there anything else which I do oversee?

Cheers,
Oliver

**Comment by graciousgrey** [comment, marked as answer]
PhotoPrism classifies very short videos as live photos.

We already have a related feature request: https://github.com/photoprism/photoprism/issues/1110
